### PR TITLE
Installed simplecov, codeclimat setup, added badges to readme.

### PR DESCRIPTION
**Pull request by f1nwe**
- [x] @Kmalynss 
- [x] @denys281 
- [x] @arsenlol 
Installed simplecov.
Added badges to readme file.
CodeClimat setup.
Configured .travis.yml to send reports from codeclimate.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
+require 'simplecov'
+SimpleCov.start do
+  add_filter '/vendor/'
+  add_filter '/lib/myfile.rb'
```
Comment (igorkasyanchuk) [inline spec/spec_helper.rb]: maybe just add all files in /vendor and /lib?

Comment (Kmalynss) [inline spec/spec_helper.rb]: @f1nwe I agree with @igorkasyanchuk, why did you add ```'/lib/myfile.rb'``` ? and not just ```/lib```? Also are you sure that absolute path should be housed here?

**Comment by Kmalynss** [comment]
👍
